libs/chain-signatures/contract/src/primitives/key_state.rs

```diff
@@ -135,6 +135,11 @@ impl Keyset {
             .key
             .clone())
     }
+
+    #[cfg(feature = "dev-utils")]
```

Comment:
I see we have both the `dev-utils` and `test-utils` feature flags. What is the distinction between the two? I'd advocate for just having a single `test-utils`[^1] feature flag for any logic that's only relevant for testing.

[^1]: I'm more used to the `test-utils` name than `dev-utils` for this purpose, but have no strong opinion on which one we go with.

Reply:
Note: This should be done as a follow-up.

Reply:
`test-utils` is for tests, whereas `dev-utils` is for production dependencies in the MPC node.

Reply:
Okay fair enough. Feels like that code should belong in the MPC node and not the contract then, but okay I can live with this.